pkg/cli/cmd/templates.go

```diff
@@ -43,16 +45,106 @@ opa:
 `
 
 const templatePreamble = `---
+# compatible config version
+version: 1 
+
 logging:
   prod: true
   log_level: info
 
 directory:
   db_path: ${TOPAZ_DIR}/db/directory.db
   seed_metadata: {{ .SeedMetadata }}
-  
+
+# remote directory is used to resolve the identity for the authorizer.
+remote_directory:
+  address: "0.0.0.0:9292" # set as default, it should be the same as the reader as we resolve the identity from the local directory service.
+  insecure: true
+
+# default jwt validation configuration
+# jwt:
+#   acceptable_time_skew_seconds: 5
+
 api:
+  reader:
+    grpc:
+      listen_address: "0.0.0.0:9292"
+      # if certs are not specified default certs will be generate with the format reader_grpc.*
+      certs:
+        tls_key_path: "${TOPAZ_DIR}/certs/grpc.key"
+        tls_cert_path: "${TOPAZ_DIR}/certs/grpc.crt"
+        tls_ca_cert_path: "${TOPAZ_DIR}/certs/grpc-ca.crt"
+    gateway:
+      listen_address: "0.0.0.0:9393"
+      # allowed_origins include localhost by default
```

Comment:
Should this be `includes` instead of `include`, assuming we are relaying that the allowed_origins always includes localhost? 

Do we include others besides localhost?

Reply:
actually the comment here is wrong. If nothing is specified in the allowed origins sections the service hosts adds by default localhost https://github.com/aserto-dev/service-host/blob/main/defaults.go#L10

However as we include this in the config template, localhost is not in the allowed origins list now.
````
- https://*.aserto.com
- https://*aserto-console.netlify.app
```

I'll update the template to add localhost to the generated configuration.